frontend/src/components/print/PrintConfigurator.vue

```diff
@@ -6,7 +6,7 @@
 
       <v-container>
         <v-row>
-          <v-col cols="12" lg="4">
+          <v-col cols="8">
```

Comment:
I don't understand why, but on screen widths <600px, the preview tabs are displayed side-by-side with the config. This is definitely not what we want. On small screens even more than on large ones, we should display them below each other.

Reply:
Fixed with https://github.com/ecamp/ecamp3/pull/2562/commits/365bfb1afa6eac1d9128ff78f344f98b782e983e
Some global styling was messing up with v-container